WebHostLib/tracker.py

```diff
@@ -1518,3 +1563,8 @@ def attribute_item(team: int, recipient: int, item: int):
     "Super Metroid": __renderSuperMetroidTracker,
     "Starcraft 2 Wings of Liberty": __renderSC2WoLTracker
 }
+
+multi_trackers: typing.Dict[str, typing.Callable] = {
+    "A Link to the Past": get_LttP_multiworld_tracker,
+    "Factorio": get_Factorio_multiworld_tracker,
+}
```

Comment:
No newline at end of file. Not sure if applying the suggestion will correctly fix it though
```suggestion
}

```